### ISSUE TITLE
Complete revamp of https://ethereum.org/en/developers/docs/data-and-analytics/

### ISSUE THREAD

**Issue by andrewhong5297** [ethereum/ethereum-org-website#9144]
### Is your feature request related to a problem? Please describe.

I have written multiple data landscape guides over the years.

[2022](https://ath.mirror.xyz/w2cxg5OP1OEcqvSgsEjSSyKRJhPmam0w-fXGogiG-8g)
[2023](https://web3datadegens.substack.com/p/2023-guide-to-web3-data-tools)

There have been great reactions from the community [to my guides](https://twitter.com/andrewhong5297/status/1611392835898449920), including being featured on The Daily Gwei.

### Describe the solution you'd like

I want to revamp the data analytics page following the utility stack model in my 2023 guide and the section descriptions. I prefer the word-for-word tool descriptions to stay in my blog, as I don't want to speak too much for others. I work at Dune, so I'm okay with leaving the Dune section in there as-is. If other tools want to add a description, they can do so in a future PR.

### Describe alternatives you've considered

My 2022 guide was more data engineering/full-stack approach. Data engineering (indexers/APIs) should be a separate page in the future.

**Comment by samajammin** [comment]
Thanks for this!

>  I prefer the word-for-word tool descriptions to stay in my blog, as I don't want to speak too much for others. I work at Dune, so I'm okay with leaving the Dune section in there as-is. If other tools want to add a description, they can do so in a future PR.

I'm not opposed to this but I personally think your descriptions of projects are pretty solid:
https://web3datadegens.substack.com/p/2023-guide-to-web3-data-tools

I'd lean towards including them. If projects decide they'd like to re-word any descriptions, they can do that but in the meantime I suspect having descriptions would increase value to visitors.

**Comment by corwintines** [comment]
Hey @andrewhong5297!

Read your articles, they were very interesting!

This sounds great, ill assign this to you to work on. As @samajammin mentioned, I think your product descriptions are good here. If we need to reword anything we can do that during the review :)

Let me know if you have any questions!

**Comment by corwintines** [comment]
I removed the stale tag as im still quite interested in this @andrewhong5297. Let me know if you are still interested in contributing this, or if there is something I can do to facilitate this :)

**Comment by andrewhong5297** [comment]
> I removed the stale tag as im still quite interested in this @andrewhong5297. Let me know if you are still interested in contributing this, or if there is something I can do to facilitate this :)

Yeah would love to! Think I was waiting on permission to get started on this.

**Comment by corwintines** [comment]
This is still something i would love to see @andrewhong5297. You have my permission to get going on this :) Would you like a hand with the outline or anything?

**Comment by andrewhong5297** [comment]
> This is still something i would love to see @andrewhong5297. You have my permission to get going on this :) Would you like a hand with the outline or anything?

yes! I'm down to revamp [this page](https://github.com/ethereum/ethereum-org-website/blob/dev/public/content/developers/docs/data-and-analytics/index.md). Will it be alright for me to call out [my newsletter](https://read.cryptodatabytes.com/) and [my community](https://read.cryptodatabytes.com/p/64a41653-deeb-45a1-bfea-d025b866bcab) near the top of the page?

**Comment by corwintines** [comment]
We usually just have  a section for further reading which links out to resources. I think in that section would be great.

**Comment by andrewhong5297** [comment]
Alright that works for me. I'll make a PR soon.

**Comment by andrewhong5297** [comment]
I've made an initial translation from my guide into here, and moved tool descriptions to a new section. 

I've removed most of my more opinionated language, but there may still be some opinions in there that you all want to remove. 

https://github.com/ethereum/ethereum-org-website/pull/13352

**Comment by andrewhong5297** [comment]
Hello?

**Comment by nloureiro** [comment]
Closing during a backlog sweep: this revamp proposal was built around the 2022/23 data-tooling landscape, which has moved on considerably, and there's been no activity toward a PR. A fresh proposal grounded in the current /developers/docs/data-and-analytics page would be a better starting point — related refinement work is also being discussed in newer issues.
